privacyidea/lib/eventhandler/usernotification.py

```diff
@@ -407,6 +411,7 @@ def do(self, action, options=None):
                                    tokenowner=tokenowner,
                                    serial=serial,
                                    tokentype=tokentype,
+                                   tokendescription=tokendescription,
```

Comment:
Please also add the `{tokendescription}` tag to the list of possible tags in the documentation here:
https://github.com/privacyidea/privacyidea/blob/ade6a2c85d34691ff5f1fa5062ac6c87d84aff8c/doc/eventhandler/usernotification.rst?plain=1#L152